src/client/actions/ActionsManager.js

```diff
@@ -1,52 +1,20 @@
 'use strict';
 
+const fs = require("fs");
+
 class ActionsManager {
   constructor(client) {
     this.client = client;
 
-    this.register(require('./MessageCreate'));
-    this.register(require('./MessageDelete'));
-    this.register(require('./MessageDeleteBulk'));
-    this.register(require('./MessageUpdate'));
-    this.register(require('./MessageReactionAdd'));
-    this.register(require('./MessageReactionRemove'));
-    this.register(require('./MessageReactionRemoveAll'));
-    this.register(require('./MessageReactionRemoveEmoji'));
-    this.register(require('./ChannelCreate'));
-    this.register(require('./ChannelDelete'));
-    this.register(require('./ChannelUpdate'));
-    this.register(require('./GuildDelete'));
-    this.register(require('./GuildUpdate'));
-    this.register(require('./InteractionCreate'));
-    this.register(require('./InviteCreate'));
-    this.register(require('./InviteDelete'));
-    this.register(require('./GuildMemberRemove'));
-    this.register(require('./GuildMemberUpdate'));
-    this.register(require('./GuildBanAdd'));
-    this.register(require('./GuildBanRemove'));
-    this.register(require('./GuildRoleCreate'));
-    this.register(require('./GuildRoleDelete'));
-    this.register(require('./GuildRoleUpdate'));
-    this.register(require('./PresenceUpdate'));
-    this.register(require('./UserUpdate'));
-    this.register(require('./VoiceStateUpdate'));
-    this.register(require('./GuildEmojiCreate'));
-    this.register(require('./GuildEmojiDelete'));
-    this.register(require('./GuildEmojiUpdate'));
-    this.register(require('./GuildEmojisUpdate'));
-    this.register(require('./ThreadCreate'));
-    this.register(require('./ThreadDelete'));
-    this.register(require('./ThreadListSync'));
-    this.register(require('./ThreadMemberUpdate'));
-    this.register(require('./ThreadMembersUpdate'));
-    this.register(require('./GuildRolesPositionUpdate'));
-    this.register(require('./GuildChannelsPositionUpdate'));
-    this.register(require('./GuildIntegrationsUpdate'));
-    this.register(require('./WebhooksUpdate'));
-    this.register(require('./TypingStart'));
-    this.register(require('./StageInstanceCreate'));
-    this.register(require('./StageInstanceUpdate'));
-    this.register(require('./StageInstanceDelete'));
+    let allFiles = fs.readdirSync("./").filter(
+        file => !["Action.js","ActionsManager.js"].some(
+            blacklist => blacklist === file
+        )
+    );
+
+    for (let file of allFiles) {
```

Comment:
```suggestion
    for (const file of allFiles) {
```

Reply:
why?

Reply:
You're aren't ever updating the value of `file`, so `const` should be preferred

Reply:
Never thought of it that way, so sure.

Reply:
https://github.com/discordjs/discord.js/pull/6102/commits/a0a567cd5972de9b0b32103f35e7bb83ad47fb36